prime-router/docs/design/design/HL7v2-FHIR-Inventory.md

```diff
@@ -0,0 +1,39 @@
+# HL7v2 to FHIR Inventory
+
+Generally, all the mappings follow
+the [inventory](https://docs.google.com/spreadsheets/d/1PaFYPSSq4oplTvw_4OgOn6h2Bs_CMvCAU9CqC4tPBgk/edit#gid=484860251)
+specified here. This mapping is comprehensive for mapping from HL7 to FHIR, but it is not a lossless process and the
+inventory does not specify how to map from FHIR to HL7.
+
+## Translating from FHIR to HL7
+
+In many cases, the FHIR bundle created from strictly following the inventory does not have enough information to
```

Comment:
```suggestion
In many cases, the FHIR bundle created by strictly following the inventory does not have enough information to
```
nit: it took me a couple reads of this sentence to understand. Maybe my brain is just tired though lol